modules/data-loading/partials/cloud/cloud-data-source-details.adoc

```diff
@@ -15,9 +15,40 @@ See Using IAM Roles for Amazon EC2 for more information.
 
 [source,json]
 {
-"type": "s3",
-"access.key": "<access key>",
-"secret.key": "<secret key>"
+    "type": "s3",
+    "access.key": "<access key>",
+    "secret.key": "<secret key>"
+}
+
+For temporary access to a bucket, we support AWS temporary credential provider. A temporary credential can be generated from AWS console, for example, using cloud shell:
+[source,bash]
+aws sts assume-role --role-arn arn:aws:iam::<account-id>:role/<role> --role-session-name "<session-name>"
+
+For more details on temporary credentials, please check https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[AWS doc].
+
+To use the temporary credential, add the below configs in the data source definition:
+[source,json]
+{
+    "type": "s3",
+    "file.reader.settings.fs.s3a.aws.credentials.provider": "org.apache.hadoop.fs.s3a.TemporaryAWSCredentialsProvider",
+    "file.reader.settings.fs.s3a.access.key": "The <AccessKeyId> in credential",
+    "file.reader.settings.fs.s3a.secret.key": "The <SecretAccessKey> in credential",
```

Comment:
Should those two keys be `access.key` and `secret.key`? I think GSQL has a precheck that requires these two entries.

Reply:
Good catch! Both will work because GSQL will do a conversion here: https://github.com/tigergraph/gle/blob/002478095e72df3e34338b8b1189d6bbaf0ccf41/gsql-server/src/main/java/com/tigergraph/schema/topology/DataSource.java#L535

I will update to access.key and secret.key